testdata/controllers/config.yaml

```diff
@@ -0,0 +1,23 @@
+defaultResync: 30s
+# by default, Iter8 controller is namespace scoped
+# clusterScoped: true
+resourceTypes:
```

Comment:
This needs to include the resource type `cm`. See PR on the branch.

Reply:
Fixed.